templates/components/entry.html.twig

```diff
@@ -176,16 +176,24 @@
                                     </li>
                                 {% endif %}
 
+                                <li class="dropdown__separator"></li>
                                 <li>
-                                    <a data-action="clipboard#copy"
-                                       href="{{ entry_url(entry) }}">{{ 'copy_url'|trans }}</a>
+                                    <a target="_blank"
+                                       href="{{ entry.apId ?? path('ap_entry', {magazine_name: entry.magazine.name, entry_id: entry.id}) }}">
+                                        {{ 'open_url_to_fediverse'|trans }}
+                                    </a>
                                 </li>
                                 <li>
                                     <a data-action="clipboard#copy"
                                        href="{{ entry.apId ?? path('ap_entry', {magazine_name: entry.magazine.name, entry_id: entry.id}) }}">
                                         {{ 'copy_url_to_fediverse'|trans }}
                                     </a>
                                 </li>
+                                <li>
+                                    <a data-action="clipboard#copy"
+                                       href="{{ entry_url(entry) }}">{{ 'copy_url'|trans }}</a>
+                                </li>
+                                <li class="dropdown__separator"></li>
```

Comment:
I would move this below the `is_granted` for each file, I don't think it's needed if there aren't more entries, the end of the list is a decent enough separator

Reply:
You are right!

Reply:
but uh.. I need to check now on if it's `is_granted` for edit OR `app.user and entry.isAuthor(app.user)` OR `is_granted` for moderator... Since you just want to add 1 separator here.

Reply:
`if is_granted('edit', entry) or if app.user and entry.isAuthor(app.user) or is_granted('moderate', entry)`  wonderful

Reply:
Done